### PR TITLE
fixing CI for pluto tutorials

### PR DESCRIPTION
**Pull request by CarloLucibello**

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -87,11 +88,11 @@ makedocs(;
...
             #  "Tutorials" => tutorial_menu,
-            #  "Tutorials" => 
-            #     [
-            #         "Intro to Graph Neural Networks" => "gnn_intro_pluto.md",
-            #         "Graph Classification" => "graph_classification_pluto.md",
-            #     ],
+             "Tutorials" => 
+                [
+                    "Intro to Graph Neural Networks" => "gnn_intro_pluto.md",
+                    "Graph Classification" => "graph_classification_pluto.md",
```
Comment (rikhuijzer) [inline docs/make.jl]:
```suggestion
                    "Intro to Graph Neural Networks" => "tutorials/gnn_intro_pluto.md",
                    "Graph Classification" => "tutorials/graph_classification_pluto.md",
```

**Comment by rikhuijzer** [comment]
@CarloLucibello In `graph_classification_pluto`, don't forget to set
```julia
dataset = let
	ENV["DATADEPS_ALWAYS_ACCEPT"] = "true"
	TUDataset("MUTAG")
end
```
without that, the build fails locally here so probably also in CI

File: docs/src/tutorials/gnn_intro_pluto.jl [hunk 2]
```diff
@@ -4,6 +4,22 @@
...
+        PackageSpec(; path=joinpath(@__DIR__,"..","..","..")),
+        PackageSpec(; name="Flux", version="0.13"),
+		PackageSpec(; name="MLDatasets", version="0.7"),
+		PackageSpec(; name="GraphMakie"),
+		PackageSpec(; name="Graphs"),
+		PackageSpec(; name="CairoMakie"),
+		PackageSpec(; name="PlutoUI"),
+    ]
+    Pkg.add(packages)
+end
```
Comment (CarloLucibello) [inline docs/src/tutorials/gnn_intro_pluto.jl:21]: @rikhuijzer a problem with this is that I get a very long terminal output in the notebook that I don't know how to suppress

Comment (CarloLucibello) [reply docs/src/tutorials/gnn_intro_pluto.jl:21]: maybe I just add all those packages to `docs/Project.toml` and activate that?

Comment (rikhuijzer) [reply docs/src/tutorials/gnn_intro_pluto.jl:21]: In Pluto you mean right? In the few latest versions you can disable console output per cell. Select it at the top right of the cell. It remembers your choice also between runs

**Comment by CarloLucibello** [comment]
hurrah!

File: docs/src/tutorials/gnn_intro_pluto.jl (hunk 2 above)
Comment (rikhuijzer) [reply docs/src/tutorials/gnn_intro_pluto.jl:22]: And if you want to hide the code for a cell, use `# hideall` like in Documenter:

```julia
# hideall
x = 1
```
will show only the output block containing 1. To hide code and output, use the semicolon.

**Comment by rikhuijzer** [comment]
> hurrah!

:partying_face: Looks like it works.

Can I add `GraphNeuralNetworks.jl` to the list of packages using `PlutoStaticHTML.jl` for documentation (https://huijzer.xyz/PlutoStaticHTML.jl/dev/#Documenter.jl)? You're also welcome to open a PR if you want

File: docs/src/tutorials/gnn_intro_pluto.jl (hunk 2 above)
Comment (CarloLucibello) [reply docs/src/tutorials/gnn_intro_pluto.jl:22]: what happens with the visibility toggle in Pluto frontend? Isn't the visibility state stored in the cell metadata?

Comment (rikhuijzer) [reply docs/src/tutorials/gnn_intro_pluto.jl:22]: Yep nowadays it is. I have to implement reading that still. Added it to my task list a few minutes ago (https://github.com/rikhuijzer/PlutoStaticHTML.jl/issues/119)

Comment (rikhuijzer) [reply docs/src/tutorials/gnn_intro_pluto.jl:22]: > Isn't the visibility state stored in the cell metadata?

I've tested this now but apparently not no in 0.19.5

**Pull request merged**